main_test.go

```diff
@@ -173,6 +173,48 @@ func TestIsStringFoundInOutput_timeOutMessageUITest(t *testing.T) {
 	}
 }
 
+func TestIsStringFoundInOutput_earlyUnexpectedExit(t *testing.T) {
+	// load sample logs 1
```

Comment:
fix in https://github.com/bitrise-io/steps-xcode-test/pull/61/commits/df48155c709bd97804449c599059c4951a801ae2